### PR TITLE
Strike Team Revamp part 1: Headsets

### PR DESCRIPTION
**Pull request by DeityLink**
- ERT and Deathsquad now have their own headsets with their own encryption keys
- Deathsquad members can talk on their channel with ":0"
- ERT members can talk on their channel with ":r"

also a few sprites that'll be relevant to the next parts of the revamp

### CODE REVIEW

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -216,10 +216,21 @@
...
+
+	New()
+		keyslot2 = new /obj/item/device/encryptionkey/deathsquad
+		..()
+
 /obj/item/device/radio/headset/ert
 	name = "CentCom Response Team headset"
-	desc = "The headset of the boss's boss. Channels are as follows: :h - Response Team :c - command, :s - security, :e - engineering, :d - mining, :q - cargo, :m - medical, :n - science."
-	icon_state = "com_headset"
+	desc = "The headset of the boss's boss. Channels are as follows: :7 - Response Team :c - command, :s - security, :e - engineering, :d - mining, :q - cargo, :m - medical, :n - science."
```
Comment (ComicIronic) [inline code/game/objects/items/devices/radio/headset.dm]: I'm not sure you should use numbers for channels, since numbers are typically languages.

Comment (DeityLink) [inline code/game/objects/items/devices/radio/headset.dm]: oh, I messed up the desc, the actual channel for the ERT is r, not 7. I think Deathsquad should keep 0 though. Every single letter of deathsquad is already a channel :/

Comment (ComicIronic) [inline code/game/objects/items/devices/radio/headset.dm]: I don't think q is.

Comment (DeityLink) [inline code/game/objects/items/devices/radio/headset.dm]: q is Diona rootspeak.

**Comment by CptWad** [comment]
:+1:

**Pull request merged**